docker/kratos/config/indentity.schema.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Quick note before I forget - there's a typo in the file name - `indentity` instead of `identity`. I'll change it later when I have the whole thing working.